pkg/gofr/crud_handlers.go

```diff
@@ -117,11 +119,12 @@ func (e *entity) Create(c *Context) (interface{}, error) {
 		fieldValues = append(fieldValues, reflect.ValueOf(newEntity).Elem().Field(i).Interface())
 	}
 
-	stmt := fmt.Sprintf("INSERT INTO %s (%s) VALUES (%s)",
+	rawStmt := fmt.Sprintf("INSERT INTO %s (%s) VALUES (%s)",
 		e.name,
 		strings.Join(fieldNames, ", "),
 		strings.Repeat("?, ", len(fieldNames)-1)+"?",
```

Comment:
Instead of rebinding the `?` for Postgres after the formation of the Query, can we change this line(125) to get the values placeholder depending on the dialect?

Reply:
Yeah, I believe we will do this, but in scope of next issue (with keywords), maybe we have to introduce functions like buildInsertQuery(dialect, table, fields) and use it in crud_handlers as well.

Reply:
Yes, it would make more sense to build queries based on the configured dialect.